java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java

```diff
@@ -61,83 +65,76 @@ public void getAccessKey_keySetInEnvironmentVariable_returnsValue() {
         assertNotEquals("", actualAccessKey);
     }
 
-    @Test
-    public void defaultConstructor_instantiated_setsConcreteDriverManager() {
-        SauceSession concreteSauceSession = new SauceSession();
-        assertTrue(concreteSauceSession.getSauceDriver() instanceof SauceDriverImpl);
-    }
-
     @Test
     public void startSession_setsBrowserKey() {
+        sauce.start();
+
         String expectedBrowserCapabilityKey = "browserName";
         String actualBrowser = sauce.getCurrentSessionCapabilities().getCapability(expectedBrowserCapabilityKey).toString();
         assertNotEquals("", actualBrowser);
     }
 
     @Test
     public void start_setsPlatformNameKey() {
+        sauce.start();
+
         String correctPlatformKey = "platformName";
         String browserSetInSauceSession = sauce.getCurrentSessionCapabilities().getCapability(correctPlatformKey).toString();
         assertEquals("Windows 10", browserSetInSauceSession);
     }
 
     @Test
     public void defaultBrowserIsLatest() {
+        sauce.start();
+
         String correctKey = "browserVersion";
         String browserSetThroughSauceSession = sauce.getCurrentSessionCapabilities().getCapability(correctKey).toString();
         assertEquals("latest", browserSetThroughSauceSession);
     }
 
     @Test
     public void defaultIsChrome() {
+        sauce.start();
+
         String actualBrowser = sauce.getCurrentSessionCapabilities().getBrowserName();
         assertEquals("chrome", actualBrowser);
     }
 
     @Test
     public void defaultIsWindows10() {
+        sauce.start();
+
         String actualOs = sauce.getCurrentSessionCapabilities().getPlatform().name();
         assertEquals("WIN10", actualOs);
     }
 
     @Test
     public void sauceOptions_defaultConfiguration_setsSauceOptions() {
+        sauce.start();
+
         MutableCapabilities sauceOptions = (MutableCapabilities) sauce.getCurrentSessionCapabilities().getCapability("sauce:options");
         String accessKey = (String) sauceOptions.getCapability("accessKey");
         assertEquals("You need to have Sauce Credentials set (SAUCE_USERNAME, SAUCE_ACCESSKEY) before this unit test will pass", "accessKey", accessKey);
     }
 
     @Test
     public void sauceOptions_startWithChrome_startsChrome() {
-        dummyRemoteDriver = mock(SauceRemoteDriver.class);
         options = new SauceOptions();
         options.withChrome();
 
-        sauce = new SauceSession(options, dummyRemoteDriver, dummyEnvironmentManager);
+        sauce = Mockito.spy(new SauceSession(options, dummyEnvironmentManager));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Mockito.spy' due to existing static import 'org.mockito.Mockito.*'](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4730723)